R/advs.R

```diff
@@ -1,4 +1,4 @@
-#' Dataset advs
+#' Vital Signs Analysis
```

Comment:
General comment: sometimes you use "Dataset" (ie Vital Signs Analysis Dataset) and sometimes not (Vital Signs Analysis). Wondering if we can choose one approach here? Maybe just always omit? What do you think?

Reply:
I agree that omitting it would look much better aesthetically, I'll make that update.